root/admin/EditBanner.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormRowTextArea from '../components/FormRowTextArea';
+import FormSubmit from '../components/FormSubmit';
+import Layout from '../layout';
+import {addColon, l} from '../static/scripts/common/i18n';
+
+type Props = {|
+  +form: FormT<{|
+    +message: FieldT<string>,
+  |}>,
+|};
+
+const EditBanner = ({form}: Props) => (
+  <Layout fullWidth title={l('Edit Banner Message')}>
+    <div id="content">
+      <h1>{l('Edit Banner Message')}</h1>
```

Comment:
Very minor, but I'd much rather this was "Edit banner message" and we started moving towards consisting mostly-lowercase headers

Reply:
Would you be kind enough to write a small guide for future contributors? :pray: 
Should it be part of the design system or something?

Reply:
Fixed in cf065e6b92b7f799120e06f822f2d16b4c9c38a8

Reply:
I'm not sure how the design system should be :) But I'm happy to make my pet peeves official, sure :D Just let me know how.